cutscene_engine/cutscene_scenes/cutscene_test.gd

```diff
@@ -0,0 +1,26 @@
+extends Node2D
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	var dialogue_starter = [$TileMap/DialogueStarter, $TileMap/DialogueStarter2]
+	
+	#func load_dialogue(file_path) -> Dictionary:
+	"""
+	Parses a JSON file and returns it as a dictionary
+	"""
+	
+	var file = File.new()
+	assert (file.file_exists("res://cutscene_engine/data/test_stroheim_dialogue1.json"))
+	
+	file.open("res://cutscene_engine/data/test_stroheim_dialogue1.json", file.READ)
+	var dialogue = parse_json(file.get_as_text())
+	assert (dialogue.size() > 0)
+	
+	for box in dialogue_starter:
+		box.load_dialogue(dialogue)
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+#func _process(delta):
+#	pass
```

Comment:
Is this file still needed?

Reply:
yes, its greatly important. How the system works is that it loads a single .json file per scene. So if you have say, level1.tscn, level2.tscn, etc. each scene would need cutscene_test in order for the whole thing to work. The title of the script is a bit of an oversight when it was being worked on, but it can be renamed to something like cutscene.gd or cutscene_load.gd or something else. This I can update accordingly

Reply:
Where is this file used exactly? We can extract this logic to somewhere which makes more sense, or at least rename it to something easier to understand.

Reply:
This could possibly be turned into a node named cutscene_base and be attached to levels that need it.